### ISSUE TITLE
SOX Dialog Feature Search

### ISSUE THREAD

**Issue by shu8** [soscripted/sox#36]
**Installed Version: 1.0.3DEV  |  Environment: Tampermonkey/Chrome**

How do you like this:

![this](https://cloud.githubusercontent.com/assets/8850830/14254904/afbe2ad8-fa89-11e5-8e36-150ccf1f4c32.gif)

? 

Thought I should ask before I add it to the settings dialog box...

Thoughts?

**Comment by enki-code** [comment]
Yes. But, let's put it in it's own defined section, not under appearance :)  
The markup is

```
<div class="header">
    <h3>Search</h3>
</div>
<div class="modal-content">
    /* search box here */
</div>

```

**Comment by shu8** [comment]
@enki-code sure I'll try that now

**Comment by shu8** [comment]
implemented in v1.0.3
